wolfgang_webots_sim/src/wolfgang_webots_sim/webots_supervisor_controller.py

```diff
@@ -122,8 +176,12 @@ def reset(self, req=None):
         self.supervisor.simulationResetPhysics()
         return EmptyResponse()
 
+    def reset_robot_init(self, name="amy"):
+        self.robot_nodes[name].loadState('__init__')
+        self.robot_nodes[name].resetPhysics()
+
     def set_initial_poses(self, req=None):
-        self.reset_robot_pose_rpy([-1, 3, 0.42], [0, 0.24, -1.57], name="amy")
+        self.reset_robot_pose_rpy([0, 0, 0.42], [0, 0, 0], name="amy")
```

Comment:
```suggestion
        self.reset_robot_pose_rpy([0, 0, 0.42], [0, 0, 0], name="amy")
```
This should probably be changed back?